src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -41,8 +41,9 @@ export const Configurator = ({
   const languages = type?.supportedLanguages(baseDataset) || [];
 
   const [dataset, setDataset] = useState<Dataset>(baseDataset);
-  const [timeFields, setTimeFields] = useState<DatasetField[]>();
+  const [timeFields, setTimeFields] = useState<DatasetField[]>([]);
   const [timeFieldName, setTimeFieldName] = useState<string | undefined>(dataset.timeFieldName);
+  const noTimeFilter = 'I dont want to use the time filter';
```

Comment:
should be able to pass in double quotes for `I don't`

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/extract_time_fields.ts#L59